images/bridge2/cmd/bridge/main.go

```diff
@@ -275,6 +275,24 @@ func sessionUpdateHandler(ctx context.Context, sess *Session) <-chan struct{} {
 	return ch
 }
 
+func sessionEventStream(ctx context.Context, sess *Session) <-chan tracks.Event {
+	ch := make(chan tracks.Event)
+	h := tracks.HandlerFunc(func(e tracks.Event) {
+		if e.Type == "audio" {
+			// if this is a transient event like "audio" we don't want to send
+			return
+		}
+		ch <- e
```

Comment:
Hmm, this is probably missing a select on ctx.Done() as well to stop sending when the context is cancelled.